src/components/workspace-utils.js

```diff
@@ -47,10 +47,17 @@ export const useWorkspaces = () => {
     const ws = await ajax(signal).Workspaces.list([
       'accessLevel',
       'public',
-      'workspace',
       'workspace.attributes.description',
       'workspace.attributes.tag:tags',
-      'workspace.workspaceVersion',
+      'workspace.authorizationDomain',
+      'workspace.cloudPlatform',
+      'workspace.createdBy',
+      // 'workspace.isLocked', // do we actually need this?
+      'workspace.lastModified',
+      'workspace.name',
+      'workspace.namespace',
+      'workspace.workspaceId',
+      // 'workspace.workspaceVersion', // do we actually need this?
```

Comment:
We probably don't need `workspaceVersion`-- I added it when we we had some front-end code for PPW migration. If you remove it, please also remove the places where it appears in unittest fixtures.

I do think we use isLocked.

In any case, what is the actual size savings for this?

Reply:
@cahrens I will address these if I continue with this! This PR is rather experimental right now, I should have been clearer about that in the description